enterprise/management/commands/remove_expired_pending_group_memberships.py

```diff
@@ -0,0 +1,46 @@
+"""
+Management command for ensuring any pending group membership, ie memberships associated with a pending enterprise user,
+are removed after 90 days.
+"""
+
+import logging
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+
+from enterprise.models import EnterpriseCustomer, EnterpriseGroupMembership
+from enterprise.utils import localized_utcnow
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command for ensuring any pending group membership, ie memberships associated with a pending enterprise
+    user, are removed after 90 days.
+
+    Example usage:
+        $ ./manage.py remove_expired_pending_group_memberships
+    """
+    help = 'Removes pending group memberships if they are older than 90 days.'
+
+    def add_arguments(self, parser):
+        parser.add_argument("-e", "--enterprise_custom")
+
+    def handle(self, *args, **options):
+        queryset = EnterpriseGroupMembership.objects.all()
+        if enterprise_arg := options.get("enterprise_customer"):
```

Comment:
Are we documenting the optional `enterprise_customer` argument somewhere?

Reply:
uhhh Let me add it to the doc string, it's not necessary for the overnight job that's to run however I liked the added flexibility for other users down the line

Reply:
updated the doc string- I'll write a quick unit test for it just to be certain